src/wallet/api/walletApi.ts

```diff
@@ -247,6 +249,11 @@ const walletApi = {
       return response.data;
     }
 
+    const message = get(response, 'data.message', '');
+    if (message === 'Transaction not found') {
```

Comment:
Maybe create a method to do this validation, or at least save the string in a variable? In case we change the error message in the future we would need to change in 3 places now.

Reply:
Created a guard method in the last commits, can you check?